src/Kinepolis/KinepolisService.php

```diff
@@ -228,4 +240,14 @@ private function uploadImage(string $token, ParsedMovie $parsedMovie, string $ev
         );
         return new AddImage($eventId, $imageId);
     }
+
+    private function linkToProduction(string $title): ProductionId
```

Comment:
This searches the production rather than doing the actual linking which happens with the command.